train/tr10-13B-ml/tr10-13B.slurm

```diff
@@ -165,7 +166,7 @@ export CMD=" \
     --load $CHECKPOINT_PATH \
     --data-path $DATA_PATH \
     --data-impl mmap \
-    --split 900,100,0 \
+    --split 950,50,0 \
```

Comment:
currently using a small dataset, so I had to give valid a larger chunk. But for the real training this needs to be restored to the above split.